CHANGELOG.rst

```diff
@@ -20,6 +20,8 @@ This document describes changes between each past release.
 - Server now returns 415 error response if client cannot accept JSON response (#461, mozilla-services/cliquet#667)
 - Server now returns 415 error response if client does not send JSON request (#461, mozilla-services/cliquet#667)
 - Add the ``__lbheartbeat__`` endpoint, for load balancer membership test.
+- Added the ``flush_endpoint``, ``schema`` and ``default_bucket`` capabilities
```

Comment:
I'm not sure this is part of the protocol (the protocol keeps unchanged here, just more capabilities have been added to it)